_posts/2022-06-08-ask-the-hybrid-brains.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Ask the Hybrid Brains, Spring 2022"
+summary: "Our first in-person hybrid Ask The Brains event"
+date: 2022-06-08T19:15:00
+meetup: TODO
+speakers:
+- name: Everyone
+image:
+  url: /img/talks/hybrid-brains.jpg
+  title: Ask the Hybrid brains
+tags:
+- workshop
+- questions
+- collaboration
+- virtual
+layout: event.html
+collection: events
+online: true
+venue:
+  name: PLATF9RM
+  link: https://www.platf9rm.com/
+  location: https://goo.gl/maps/pkoNTTRA27bh5GKD8
+  address: Floor 5, Tower Point, 44 North Road, Brighton, BN1 1YR
+  latlong: 50.825944,-0.1437302
+# draft: true # Delete this line to publish
+---
+We're back! You will be happy to know that you will finally have the opportunity to come face-to-face with like-minded developers and enthusiasts (at Async at least) for the first time in over 2 years! Can't make it in-person? Don't worry, you'll also be able to join us online.
+
+I am very happy to announce that on the evening of Wednesday 8th June, we will be hosting an in-person hybrid version of our popular __"Ask the Brains"__ event for the first time. This means that you will have the ability to join us in-person at [PLATF9RM](https://goo.gl/maps/pkoNTTRA27bh5GKD8) in Brighton, or to join us online!
+
+Inspired by the monthly Brighton [ALT.NET meetup](http://brightonalt.net/), we will be asking/talking/ranting about everything web-related - 5 - 10 minutes per topic.
```

Comment:
I like "ranting" but it's maybe not so welcoming! Especially as we say it's trying to be a friendly environment in the next para